ui/src/app/views/data-page/component.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import {WorkspacesServiceStub, WorkspaceStubVariables} from 'testing/stubs/workspace-service-stub';
+
+import {DataPage} from 'app/views/data-page/component';
+
+import {registerApiClient} from 'app/services/swagger-fetch-clients';
+import {currentWorkspaceStore, urlParamsStore} from 'app/utils/navigation';
+import {ResourceCard} from 'app/views/resource-card/component';
+import {CohortsApi, ConceptsApi, ConceptSetsApi, WorkspacesApi} from 'generated/fetch';
+import {CohortsApiStub, exampleCohortStubs} from 'testing/stubs/cohorts-api-stub';
+import {ConceptsApiStub} from 'testing/stubs/concepts-api-stub';
+import {ConceptSetsApiStub} from 'testing/stubs/concept-sets-api-stub';
+import {WorkspacesApiStub} from 'testing/stubs/workspaces-api-stub';
+
+import {
+  WorkspaceAccessLevel,
+} from 'generated';
+import {waitOneTickAndUpdate} from 'testing/react-test-helpers';
+
+describe('DataPage', () => {
+  beforeEach(() => {
+    registerApiClient(CohortsApi, new CohortsApiStub());
+    registerApiClient(ConceptsApi, new ConceptsApiStub());
+    registerApiClient(ConceptSetsApi, new ConceptSetsApiStub());
+    registerApiClient(WorkspacesApi, new WorkspacesApiStub());
+    urlParamsStore.next({
+      ns: WorkspaceStubVariables.DEFAULT_WORKSPACE_NS,
+      wsid: WorkspaceStubVariables.DEFAULT_WORKSPACE_ID
+    });
+    currentWorkspaceStore.next({
+      ...WorkspacesServiceStub.stubWorkspace(),
+      accessLevel: WorkspaceAccessLevel.OWNER,
+    });
+  });
+
+  it('should render', async() => {
+    const wrapper = mount(<DataPage />);
+    await waitOneTickAndUpdate(wrapper);
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should show resource cards of all types', async() => {
```

Comment:
nit: we aren't displaying notebooks on this page (I'm assuming), so test title more accurately may say 'should show expected resource cards' or 'should show applicable concept sets and cohorts'  something like that...